clusterloader2/pkg/util/util.go

```diff
@@ -129,6 +134,28 @@ func getMap(dict map[string]interface{}, key string) (map[string]interface{}, er
 	return mapValue, nil
 }
 
+func getStringArray(dict map[string]interface{}, key string) ([]string, error) {
+	value, exists := dict[key]
+	if !exists || value == nil {
+		return nil, &ErrKeyNotFound{key}
+	}
+
+	mapValueInt, ok := value.([]interface{})
```

Comment:
Why "Int"? These aren't ints..

Did you mean int to be shortcut of internal? If so, let's make it explicit.

Also maybe just "sliceValue" or sth like that?

Reply:
Int like "Interface", renamed anyway